### DISCUSSION TITLE
MVTLayer Flickering in Example on deck.gl's Website

### DISCUSSION THREAD

**Discussion by aemonm** [visgl/deck.gl#7846] [General]
Hi,

I have an issue with my project with the MVT Layer flickering and shapes popping in and out when zooming. I noticed the same issue on the [deck.gl example page](https://deck.gl/docs/api-reference/geo-layers/mvt-layer). Is this a known issue? Is there a workaround?

**Comment by lianzhao** [comment]
I guess the reason is that you have multiple polygons at same location in a single layer. These polygons have same 'z', so they fight with each other. For example, when you have a lake and an island in that lake in a single layer, it flicks. For the solution, try split into multiple layers like 'water-layer' and 'land-layer'?
